src/identity/selectors/index.ts

```diff
@@ -52,6 +48,18 @@ export const selectOrgCreationAvailableUpgrade = (
   return state.identity.allowances.orgCreation.availableUpgrade
 }
 
+export const selectQuartzActiveOrgs = (
+  state: AppState
+): AppState['identity']['quartzOrganizations']['orgs'] => {
+  return state.identity.quartzOrganizations.orgs.filter(
+    org => org.state !== 'suspended' && org.state !== 'pending'
+  )
+}
```

Comment:
Orgs can only be `active`, `suspended`, or `pending`, so this selector could be written `org ==> org.state === 'active'`. 

The negative was adopted here to ensure that if more states are added for orgs that _should_ be shown to the user, we're being overinclusive rather than underinclusive.

Reply:
So if we add another state that is supposed to restrict the account, it might get through this check? Like if we add `delinquent` and forget to update this selector, then deliquent accounts will show up in the header?

Reply:
Correct. On balance, let's switch this back to the positive - `org => org.state === provisioned`. openAPI will identify any changes in the options for org state, and you're right that we could just as easily want to add a state that restricts access here.